inference-engine/src/vpu/common/include/vpu/ngraph/operations/exp_gather_elements.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph { namespace vpu { namespace op {
+
+class ExpGatherElements : public ngraph::op::Op {
```

Comment:
The operations' name is not the final, so I'm open for all suggestions :)
`Exp` stands for `Experimental`

Reply:
Could you please explain what is the difference between new GatherElements and other Gathers